### DISCUSSION TITLE
Mobile Margin Centering - For Different screen sizes

### DISCUSSION THREAD

**Discussion by chrisborell** [Clooos/Bubble-Card#371] [Questions about config, custom styles and templates]
My partner and I are using phones with varying screen sizes (an iPhone 15 Pro Max and an iPhone 14 Pro). We've noticed that with our current theme, the popup appears off-center. Is there a method to center the popup differently based on the screen size, or to have it automatically adjust to be centered?

I've tried the 'Margin' option but it appears off-center to one of us.

**Comment by MrBearPresident** [comment]
Did you solve this?
Can this be closed?
